src/applications/mhv/landing-page/actions/hca-es.js

```diff
@@ -0,0 +1,36 @@
+import * as Sentry from '@sentry/browser';
+import { apiRequest } from '~/platform/utilities/api';
+
+export const FETCH_HCA_ENROLLMENT_STATUS_STARTED =
+  'FETCH_HCA_ENROLLMENT_STATUS_STARTED';
+export const FETCH_HCA_ENROLLMENT_STATUS_SUCCEEDED =
+  'FETCH_HCA_ENROLLMENT_STATUS_SUCCEEDED';
+export const FETCH_HCA_ENROLLMENT_STATUS_FAILED =
+  'FETCH_HCA_ENROLLMENT_STATUS_FAILED';
+
+export const fetchHcaEnrollmentStatusStarted = () => ({
+  type: FETCH_HCA_ENROLLMENT_STATUS_STARTED,
+});
+
+export const fetchHcaEnrollmentStatusSucceeded = payload => ({
+  type: FETCH_HCA_ENROLLMENT_STATUS_SUCCEEDED,
+  payload,
+});
+
+export const fetchHcaEnrollmentStatusFailed = payload => ({
+  type: FETCH_HCA_ENROLLMENT_STATUS_FAILED,
+  payload,
+});
+
+export const fetchHcaEnrollmentStatus = () => dispatch => {
```

Comment:
Is there a way to reuse what is in `src/applications/hca/utils/actions.js` or `src/platform/user/profile/actions/hca.js`. I feel there is some duplication there.